CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## x.y.z (unreleased)
 
 ### Added
-
+- Debug mode. Throw debug exceptions only if debug is active.
```

Comment:
Could you also please link the PR (i.e., use the same format as for the other changelog entries)?